UI for movie-db

ADITIRAI290698 opened the pull request:
[Discussion](https://github.com/orgs/FifthTry/discussions/90)

ADITIRAI290698 committed:
add_movie

ADITIRAI290698 committed:
add_movie

ADITIRAI290698 committed:
added_add-movie-page

ADITIRAI290698 committed:
added_add-movie

ADITIRAI290698 committed:
Merge branch 'main' of https://github.com/FifthTry/movie-db into ui

Heulitig committed:
removed .packages/ contents

akasahi committed:
fmt

akasahi committed:
Merge branch 'main' of github.com:FifthTry/movie-db into ui

ADITIRAI290698 committed:
added_add-review

amitu commented on dj/ui/FPM.ftd:
```diff
@@ -0,0 +1,10 @@
+-- fpm.package: movie-db
+base: https://raw.githubusercontent.com/FifthTry/movie-db
+
+-- fpm.dependency: fifthtry.github.io/doc-site as ds
```
We should not use doc-site for this project. Create a custom page please. Feel free to use dependencies of doc-site, eg header, typography etc, but create custom UI.

ADITIRAI290698 committed:
updated

ADITIRAI290698 committed:
update1

amitu commented on dj/ui/add_movie.ftd:
```diff
@@ -61,6 +94,12 @@ optional string list-desc3:
 optional string list-desc4:
 optional string list-desc5:
 optional string list-desc6:
+optional string list-desc7:
+optional string list-desc8:
+optional string list-desc9:
+optional string list-desc10:
+optional string list-desc11:
+optional string list-desc12:
```
Why is this not a list?

ADITIRAI290698 replied:
will make the change.

ADITIRAI290698 committed:
update2

AviralVerma13 committed:
Updated base

AviralVerma13 committed:
Update index.ftd

AviralVerma13 committed:
Update add_movie.ftd

AviralVerma13 committed:
Update add_movie.ftd

AviralVerma13 committed:
fixed font color

AviralVerma13 committed:
Update add_review.ftd

Fixed font color and padding-bottom, Added API

AviralVerma13 committed:
setting up logo

AviralVerma13 committed:
removing header assets

AviralVerma13 committed:
calling APIS

Added all the APIS, except for throwing error for blank and inappropriate inputs, to be added if easy.
Compromised the aesthetic UI, will build soon.
Removed rating field
Calling the list in the row instead of individual calls, as per Amitu's suggestion.

AviralVerma13 committed:
minor corrections

AviralVerma13 committed:
minor fixes

AviralVerma13 committed:
Calling APIS

Added all the APIS, except for throwing error for blank and inappropriate inputs, to be added if easy.
Compromised the aesthetic UI, will build soon.
The rating field has decimal input.

AviralVerma13 committed:
Improved placeholder UI a bit

work in progress

AviralVerma13 committed:
Improved placeholder UI a bit

Work in progress

AviralVerma13 committed:
removed redundant code

AviralVerma13 committed:
removing redundant code

AviralVerma13 committed:
Corrected Heading

ADITIRAI290698 committed:
updated2 add-movie ui

ADITIRAI290698 committed:
changed file name

AviralVerma13 committed:
Rename add-movie.ftd to add_movie.ftd

AviralVerma13 committed:
Rename add-review.ftd to add_review.ftd

AviralVerma13 committed:
removed color calling from text fields

ADITIRAI290698 committed:
Added homepage

AviralVerma13 committed:
Fixed body to title

AviralVerma13 committed:
Improved UI

AviralVerma13 committed:
Create moviepage.ftd

AviralVerma13 committed:
Added pseudo API

AviralVerma13 committed:
Tried adding API for movie search

Have called the API in "moviepage"

AviralVerma13 committed:
Modify1 Next-previous

AviralVerma13 committed:
Added moviepage url

AviralVerma13 committed:
Update views.py

AviralVerma13 committed:
Added moviepage URL2

ADITIRAI290698 committed:
Added moviepage

ADITIRAI290698 committed:
updated-moviepage

ADITIRAI290698 committed:
updated_add-movie

AviralVerma13 committed:
trying iframe for poster

AviralVerma13 committed:
working poster url

AviralVerma13 committed:
improve add_movie UI width

AviralVerma13 committed:
link to home page (movies)

ADITIRAI290698 committed:
Removed comments from add-movie page

ADITIRAI290698 committed:
updated add-review page

ADITIRAI290698 committed:
Updated moviepage

AviralVerma13 committed:
link back to movies page

AviralVerma13 committed:
minor fixes

AviralVerma13 committed:
minor fix in review

AviralVerma13 committed:
iframe fix in main page

ADITIRAI290698 committed:
Updated2 moviepage

ADITIRAI290698 committed:
Updated header for homepage

AviralVerma13 committed:
All pages interlinked

AviralVerma13 committed:
Delete add_movie2.ftd

ADITIRAI290698 committed:
updated2 homepage

amitu commented:
@ADITIRAI290698, @AviralVerma13 ensure this PR is merged today. You do not have to wait for a feature to be done before you merge the PR. You block merging a PR only if you believe there would an actual problem if the PR is merged.

AviralVerma13 committed:
Delete deewar.png

AviralVerma13 committed:
Final UI homepage 1
